spec/core/ics-033-multi-hop/README.md

```diff
@@ -0,0 +1,571 @@
+---
+ics: 33
+title: Multi-hop Channel
+stage: draft
+required-by: 4
+category: IBC/TAO
+kind: instantiation
+author: Derek Shiell <derek@polymerlabs.org>, Wenwei Xiong <wenwei@polymerlabs.org>, Bo Du <bo@polymerlabs.org>
+created: 2022-11-11
+modified: 2022-12-16
+---
+
+## Synopsis
+
+This document describes a standard for multi-hop IBC channels. Multi-hop channels specifies a way to route messages across a path of IBC enabled blockchains utilizing multiple pre-existing IBC connections.
+
+### Motivation
+
+The current IBC protocol defines messaging in a point-to-point paradigm which allows message passing between two directly connected IBC chains, but as more IBC enabled chains come into existence there becomes a need to relay IBC packets across chains because IBC connections may not exist between the two chains wishing to exchange messages. IBC connections may not exist for a variety of reasons which could include economic inviability since connections require client state to be continuously exchanged between connection ends which carries a cost.
+
+### Definitions
+
+Associated definitions are as defined in referenced prior standards (where the functions are defined), where appropriate.
+
+`Connection` is as defined in [ICS 3](https://github.com/cosmos/ibc/tree/main/spec/core/ics-003-connection-semantics).
+
+`Channel` is as defined in [ICS 4](https://github.com/cosmos/ibc/tree/main/spec/core/ics-004-channel-and-packet-semantics).
+
+`Channel Path` is defined as the path of connection IDs along which a channel is defined.
+
+`Connection Hop` is defined as the connection ID of the connection between two chains along a channel path.
+
+### Desired Properties
+
+- IBC channel handshake and message packets should be able to be utilize pre-existing connections to form a logical proof chain to relay messages between unconnected chains.
+- Relaying for a multi connection IBC channel should NOT require additional writes to intermediate hops.
```

Comment:
Not sure about this as we do need client updates. 
```suggestion
- Relaying for a multi connection IBC channel should require minimal number of additional writes to state on intermediate hops.
```

Reply:
True, I think this was a statement to contrast with prior multi-hop designs which would have written ICS-04 packet data to intermediate hops. I'll update the wording to capture all the nuance.